pkg/reconcile/pipeline/infinispan/handler/provision/gossiprouter.go

```diff
@@ -67,6 +69,10 @@ func GossipRouter(i *ispnv1.Infinispan, ctx pipeline.Context) {
 
 		var addKeystoreVolume, addTruststoreVolume bool
 		if i.IsSiteTLSEnabled() {
+			// enable diagnostic for k8s probes
+			enableJgrpDiag = true
```

Comment:
yes, because I need the Gossip Router to open the port so probe can "ping" it.

Reply:
So we need to use the diagnostic port as it's a TCP probe and when TLS is enabled there's no other way for the health check to succeed on the router pod?

Reply:
The health check succeeds with TLS but the issue is the gossip router logging `javax.net.ssl.SSLHandshakeException: Remote host terminated the handshake` because the probe does not do the TLS handshake (it looks like it only tries to ping it).

With this PR, I'm enabling the diagnostic port when TLS is enabled; the health check can probe that port and we avoided the exceptions in the log.

Reply:
The exception reqported when using openssl is different until https://github.com/wildfly-security/wildfly-openssl/pull/123 is merged